src/components/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -307,7 +207,7 @@ function NetworkAreaDiagramContent(props: NetworkAreaDiagramContentProps) {
                 null,
                 isEditNadMode,
                 true,
-                dynamicCssRules,
+                [0, 2000, 3500, 6000, 9000, 12000, 15000],
```

Comment:
maybe create a constant for this array ?

Reply:
ok